guides/theming-your-components.md

```diff
@@ -1,80 +1,81 @@
-# Theming your custom components
-In order to style your own components with Angular Material's tooling, the component's styles must be defined with Sass.
+### Theming your own component the Angular Material way
+In order to style your own components with Angular Material's tooling, the component's styles must
+be defined with Sass.
 
-### Using `@mixin` to automatically apply a theme
-
-#### Why using `@mixin`
-The advantage of using a `@mixin` function is that when you change your theme, every file that uses it will be automatically updated.
-Calling the `@mixin` with a different theme argument allows multiple themes within the app or component.
-
-#### How to use `@mixin`
-We can better theme our custom components by adding a `@mixin` function to its theme file, and then call this function to apply a theme.
-
-All you need is to create a `@mixin` function in the custom-component-theme.scss
+#### 1. Define all color and typography styles in a "theme mixin" for the component
```

Comment:
This section should include examples for both theme and typography, currently we mention typography but don't show it in the examples.

Reply:
Done